File: 2 - Condições/condicoes2.c
//Escreva um programa que leia um número e mostre se ele é múltiplo de 7.

#include <stdio.h>

int main()
{
  int numero;
  scanf("%d", &numero);
  if (numero % 7 == 0)
  {
    printf("%d eh multiplo de 7.", numero);
  }
  else
  {
    printf("%d nao eh multiplo de 7.", numero);
  }

  return 0;
}

File: 2 - Condições/Java/condicoes2.java
//Escreva um programa que leia um número e mostre se ele é múltiplo de 7.

import java.util.Scanner;

public class condicoes2 {

  public static void main(String[] args) {
    System.out.println("Digite um número: ");
    Scanner scan = new Scanner(System.in);
    int numero = scan.nextInt();
    if (numero % 7 == 0) {
      System.out.println("O número é multiplo de 7");
    } else {
      System.out.println("O número não é multiplo de 7");
    }
  }
}
